src/utils/Server.ts

```diff
@@ -0,0 +1,57 @@
+import * as bodyParser from 'body-parser';
+import * as cookieParser from 'cookie-parser';
+import * as express from 'express';
+import * as session from 'express-session';
+import * as nunjucks from 'nunjucks';
+import * as path from 'path';
+import { appRouter } from '../routes/routes';
+
+export class Server {
```

Comment:
Is there a better place to put this Server class? It contains a start() method used to start the application. I see this as more of an entry point to the application rather than a Util

Reply:
What would you suggest? Putting it in `main` folder or just in the src directory?

Reply:
I would put it with the app.ts, so the src directory. If there's anything in there that you feel we would reuse in the future, then we could pull this out